### PR TITLE
Provide a sample .env and update readme

### PR DESCRIPTION
**Pull request by bparees**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [x] Documentation Update

### CODE REVIEW

**Comment by bparees** [comment]
@thoraxe ptal

File: readme.md [hunk 1]
```diff
@@ -40,24 +40,23 @@ _Below is an example of how you can instruct your audience on installing and set
...
 in order to run the API service  
    ```sh
         uvicorn ols:app --reload
    ```
 
+### Query the server
+
+To send a request to the server you can use the following curl command:
+   ```sh
+      curl -X 'POST' 'http://127.0.0.1:8000/ols' -H2 'accept: appli22cation/json' -H 'Content-Type: application/json' -d '{"query": "write a deployment yaml for the mongodb image"}'
```
Comment (vishnuchalla) [inline readme.md]: Looks like we have a type here.

Comment (bparees) [reply readme.md]: thanks, fixed

**Pull request merged**